### ISSUE TITLE
numpy: np.sum() return type incorrectly recognized as NoneType

### ISSUE THREAD

**Issue by luyiming** [PyCQA/pylint#2015]
### Steps to reproduce
1. cat bug.py

``` python
import numpy as np
a = -np.sum([1, 2, 3])
```

2. pylint bug.py

### Current behavior
```bash
E:  2, 4: bad operand type for unary -: NoneType (invalid-unary-operand-type)
```

### Expected behavior
No error at all.

### pylint --version output
```bash
pylint 1.8.2,
astroid 1.6.1
Python 3.6.4 |Anaconda, Inc.| (default, Jan 16 2018, 12:04:33)
[GCC 4.2.1 Compatible Clang 4.0.1 (tags/RELEASE_401/final)]
```

**Comment by PCManticore** [comment]
Closed as duplicate of https://github.com/PyCQA/pylint/issues/1558. This will be fixed in the next bug fix release of astroid.
